BitwardenShared/Core/Platform/Services/NotificationService.swift

```diff
@@ -279,14 +279,13 @@ class DefaultNotificationService: NotificationService {
         await stateService.setLoginRequest(data)
 
         // Get the email of the account that the login request is coming from.
-        let loginSourceAccount = try await stateService.getAccounts()
-            .first(where: { $0.profile.userId == data.userId })
-        let loginSourceEmail = loginSourceAccount?.profile.email ?? ""
+        let loginSourceAccount = try await stateService.getAccount(userId: data.userId)
+        let loginSourceEmail = loginSourceAccount.profile.email
 
         // Assemble the data to add to the in-app banner notification.
         let loginRequestData = try? JSONEncoder().encode(LoginRequestPushNotification(
             timeoutInMinutes: Constants.loginRequestTimeoutMinutes,
-            userEmail: loginSourceEmail
+            userId: loginSourceAccount.profile.userId
```

Comment:
This previously stored the email of the account that the request was for. Which would be problematic if there were multiple accounts with the same email. So I switched to the use the user ID.